requirements/examples.txt

```diff
@@ -1,2 +1,3 @@
 torchvision>=0.4.1,<0.9.0
-gym>=0.17.0
\ No newline at end of file
+gym>=0.17.0
+nvidia-dali --extra-index-url https://developer.download.nvidia.com/compute/redist
```

Comment:
we shall install it at least on our Docker image from the source
```
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/c1/fc_9d4b10c1cwmsm0l6cgzxc0000gn/T/pip-install-xo27f9qp/nvidia-dali-cuda100/setup.py", line 150, in <module>
        raise RuntimeError(open("ERROR.txt", "r").read())
    RuntimeError:
    ###########################################################################################
    The package you are trying to install is only a placeholder project on PyPI.org repository.
    This package is hosted on NVIDIA Python Package Index.
    
    This package can be installed as:
    ```
    $ pip install nvidia-pyindex
    $ pip install nvidia-dali-cuda100
    ```
    
    Please refer to NVIDIA DALI installation guide for instructions:
    https://docs.nvidia.com/deeplearning/dali/user-guide/docs/installation.html
    ###########################################################################################
```
cc: @ydcjeff

Reply:
Yeah we won't be able to include this as a requirement here, but I think I'll add a note in the docs